src/Illuminate/Database/Eloquent/Concerns/HasUuids.php

```diff
@@ -49,16 +49,10 @@ public function uniqueIds()
      */
     public function getKeyType()
     {
-        return 'string';
-    }
+        if (in_array($this->getKeyName(), $this->uniqueIds())) {
```

Comment:
```suggestion
        if (\in_array($this->getKeyName(), $this->uniqueIds())) {
```

Reply:
I might be wrong, but this method FQN will work faster.

Reply:
This is not necessary. We don't do this anywhere else.